Author: Holger Jaekel <holger.jaekel@gmx.de>
Summary: Remove warning for libspatialindex >=2.1 as patch has been applied
         in Alpine Linux
----

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -456,9 +456,6 @@
     set(SPATIALINDEX_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/external/spatialindex/include)
   else()
     find_package(Spatialindex REQUIRED)
-    if(SPATIALINDEX_VERSION VERSION_GREATER_EQUAL "2.1")
-      message(FATAL_ERROR "Cannot build QGIS using libspatialindex >= 2.1, see https://github.com/libspatialindex/libspatialindex/issues/276")
-    endif()
   endif()
   find_package(LibZip REQUIRED)
   set (WITH_INTERNAL_NLOHMANN_JSON ${PREFER_INTERNAL_LIBS} CACHE BOOL "Determines whether the vendored copy of nlohmann-json should be used")
